INSTALL.md

```diff
@@ -28,10 +28,10 @@ curl -L -o cirrus https://github.com/cirruslabs/cirrus-cli/releases/latest/downl
 
 ## Golang
 
-If you have [Go](https://golang.org/) installed, you can run:
+If you have [Go](https://golang.org/) 1.15 or newer installed, you can run:
 
 ```
-go get github.com/cirruslabs/cirrus-cli/...
+(cd && GO111MODULE=on go get github.com/cirruslabs/cirrus-cli/...)
```

Comment:
Why the `cd` is needed?

Reply:
https://dev.to/maelvls/why-is-go111module-everywhere-and-everything-about-go-modules-24k#-raw-go111module-endraw-with-go-114